### DISCUSSION TITLE
Is Go version 1.21 supported?

### DISCUSSION THREAD

**Discussion by jan-gebauer** [roblillack/mars#31] [Q&A] [answered]
Is Go version 1.21 supported? If not, what about 1.20? README says no but the latest commit suggests that yes.

**Comment by roblillack** [comment, marked as answer]
Hey @jan-polivka, thanks for pointing that out. Both, 1.20 and 1.21 are supported just fine. I just updated the README.

**Comment by jan-gebauer** [comment]
Nice, that was very prompt. Thank you very much!
